src/main/scala/rocket/PTW.scala

```diff
@@ -772,6 +773,7 @@ trait CanHavePTWModule extends HasHellaCacheModule {
   val outer: CanHavePTW
   val ptwPorts = ListBuffer(outer.dcache.module.io.ptw)
   val ptw = Module(new PTW(outer.nPTWPorts)(outer.dcache.node.edges.out(0), outer.p))
+  ptw.io.mem <> DontCare
```

Comment:
The `DontCare` connection may need chipsalliance/chisel3#2747, but it is OK for now.